remote/lib/app_preferences.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:shared_preferences/shared_preferences.dart';
+
+class AppPreferences {
```

Comment:
After what we did with rtchat I'm under the impression that it's probably just cleaner to directly read/write to shared preferences: https://github.com/muxable/muxer/blob/main/control/lib/screens/control/publish.dart#L139

This would make this wrapper class and model unnecessary.